modular-code.md

```diff
@@ -11,36 +11,49 @@ Development does not want to spend the extra resources needed to develop modular
 * Time commitments might already have been made for customer deliveries (not changeable).
 
 **Forces:**
-* There is a learning curve to writing code that can be reused.
+* There is a learning curve to writing code that can be reused. Developers might not know how to write modular code. Education might be needed.
```

Comment:
Not a huge deal in this case, but recommend [putting each sentence on its own line](https://github.com/paypal/InnerSourcePatterns/pull/96).

Reply:
Resolved.